bdk-ffi/src/bdk.udl

```diff
@@ -316,46 +316,88 @@ interface TxidParseError {
 // bdk_wallet crate - types module
 // ------------------------------------------------------------------------
 
+/// Types of keychains
 enum KeychainKind {
+  /// External keychain, used for deriving recipient addresses.
   "External",
+
+  /// Internal keychain, used for deriving change addresses.
   "Internal",
 };
 
+/// A derived address and the index it was found at.
 dictionary AddressInfo {
+  /// Child index of this address
```

Comment:
I think the standard for Rust and Kotlin/Java is to have a full stop at the end of the first sentence, even if there is only one. I noticed that it's not always respected in the bdk docs, but then added when people point them out. In case the Rust docs are faulty of this, I suggest we simply fix here for the ffi docs.

Reply:
Discussed on the Bindings Call to just re-visit this topic in a few months